crates/matrix-sdk/src/room/timeline/mod.rs

```diff
@@ -577,6 +613,14 @@ pub enum Error {
     /// The event is currently unsupported for this use case.
     #[error("Unsupported event")]
     UnsupportedEvent,
+
+    /// Couldn't read the attachment data from the given URL
+    #[error("Invalid attachment data")]
+    InvalidAttachmentData,
+
+    /// The attachment name/body is invalid
```

Comment:
Oh I see it's specified in the doc', but the name is misguiding. Maybe rename to `InvalidAttachment`, short?

Reply:
Decided to go with `InvalidAttachmentFileName` just to be extra clear